pkgs/applications/misc/flashprint/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, lib
+, fetchurl
+, dpkg
+, glibc
+, gcc-unwrapped
+, autoPatchelfHook
+, libGLU
+, qtbase
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation {
+  name = "flashprint";
+  version = "5.6.1";
+  src = fetchurl {
+    url = "https://en.fss.flashforge.com/10000/software/01e7742d5a721201cd33d4f8c083b63b.deb";
+    sha256 = "sha256-lvK4igv/L1zrqNtqbWtCyZrSZbkY8Ak/Li3NeVAH1YU=";
+  };
+
+  nativeBuildInputs = [ autoPatchelfHook dpkg wrapQtAppsHook ];
+  buildInputs = [ glibc gcc-unwrapped libGLU qtbase ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    dpkg -x $src $out
+    mv $out/usr/* $out/
+    rm -r $out/usr
+    ln -s $out/share/FlashPrint5/FlashPrint $out/bin/
+  '';
+
+  meta = with lib; {
```

Comment:
`with lib;` was only used for `licenses.unfree` so I would remove this, and write `lib.licenses.unfree` instead

Reply:
it could be used for maintainers too

Reply:
Yes, but it only you just wrote `with maintainers;` instead of `with lib.maintainers;`
I personally would just write out the whole expression, instead of using `with`, but I won't stop you, if you don't want to do that.

Reply:
hmm, putting a `with lib;` there instead writing out the lib several times feels more elegant to me even though it doesn't really decrease the amount of text.